src/gui/PlaylistWidget.cpp

```diff
@@ -660,10 +660,18 @@ QTreeWidgetItem *PlaylistWidget::newEntry(const Playlist::Entry &entry, QTreeWid
 	setEntryIcon(img, tWI);
 
 	tWI->setFlags(tWI->flags() &~ Qt::ItemIsDropEnabled);
+	if(entry.flags & Playlist::Entry::FlagSkip)
+	{
+		QFont font = tWI->font(0);
+		font.setStrikeOut(true);
+		tWI->setFont(0, font);
+	}
 	tWI->setText(0, entry.name);
 	tWI->setData(0, Qt::UserRole, entry.url);
-	tWI->setText(2, Functions::timeToStr(entry.length));
-	tWI->setData(2, Qt::UserRole, entry.length);
+	tWI->setText(1, ((entry.flags & Playlist::Entry::FlagStopAfter) ? tr("stop") : ""));
```

Comment:
Playlist shouldn't have additional column - looks bad to me. Maybe use italic font for stop-after entry? It'll also simplify everything (less changes, etc) :)